datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxBackupEndpoint.java

```diff
@@ -36,6 +40,14 @@ SdxBackupResponse backupDatalakeByName(@PathParam("name") String name,
             @QueryParam("backupLocation") String backupLocation,
             @QueryParam("backupName") String backupName);
 
+    @POST
+    @Path("{name}/backupDatalakeConditionallyCloseConnections")
```

Comment:
Please change the API to backupDatabaseInternal as this API. This will be API that will be used going

Reply:
Done.